CHANGELOG.md

```diff
@@ -4,14 +4,29 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
 
+#### Features
+
+#### Bug fixes and improvements
+
+## Mapbox Navigation SDK 2.4.0-alpha.1 - February 24, 2022
+### Changelog
+[Changes between v2.3.0-rc.3 and v2.4.0-alpha.1](https://github.com/mapbox/mapbox-navigation-android/compare/v2.3.0-rc.3...v2.4.0-alpha.1)
+
 #### Features
 - Introduced a `NavigationRoute` object and related functions like `NavigationRouterCallback`, `MapboxNavigation#setNavigationRoutes`, etc. `NavigationRoute` is a domain specific wrapper on top of `DirectionsRoute` which provides information (and enforces its presence) about the original `DirectionsResponse` that this route is part of, as well as what `RouteOptions` were used to generate it.
   Most of the Navigation SDK APIs that rely only on `DirectionsRoute` are now marked as `Deprecated` since there might be features introduced in the future that would require the `NavigationRoute` instances instead.
   There are `NavigationRoute#toDirectionsRoute()` and `DirectionsRoute#toNavigationRoute()` compatibility extensions available, however, the latter is lossy since `DirectionsRoute` cannot carry as much information as `NavigationRoute` and the recommended migration path is to use `NavigationRouterCallback` instead of the old `RouterCallback` to request routes.
   This change might require action when integrating if you have hardcoded routes in your test suites that do not have all necessary fields to support `DirectionsRoute` to `NavigationRoute` mapping (mostly `DirectionsRoute#routeIndex`) - in these cases make sure to provide the missing data or regenerate the routes using `MapboxNavigation#requestRoutes`.
   [#5411](https://github.com/mapbox/mapbox-navigation-android/pull/5411)
 
```

Comment:
We should add https://github.com/mapbox/mapbox-navigation-android/pull/5490 under `#### Bug fixes and improvements`.

Reply:
added